content/patterns/alert/alert-pattern.html

```diff
@@ -14,42 +14,35 @@
   </head>
   <body>
     <main>
-      <h1>Alert Pattern</h1>
+      <h1>アラート パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          An <a class="role-reference" href="#alert">alert</a> is an element that displays a brief, important message in a way that attracts the user's attention without interrupting the user's task.
-          Dynamically rendered alerts are automatically announced by most screen readers, and in some operating systems, they may trigger an alert sound.
-          It is important to note that, at this time, screen readers do not inform users of alerts that are present on the page before page load completes.
+          <a class="role-reference" href="#alert">アラート</a>は、利用者のタスクを中断することなく利用者の注意を引く方法によって、簡潔で重要なメッセージを表示する要素です。動的にレンダリングされたアラートは、ほとんどのスクリーンリーダーで自動的にアナウンスされ、一部のオペレーティングシステムでは、アラート音が鳴る場合があります。現時点でスクリーンリーダーは、ページの読み込みが完了する前にページ上に存在するアラートを、利用者に通知しないことに注意することが重要です。
```

Comment:
「注意を引く方法によって」は日本語としてこなれていない感があるので、もう少し柔らかめに意訳してもいいかもしれません。「注意を引くことで」「注意を引くように」くらいでどうでしょうか。
いまいち自信がありませんが、文脈的に、without... 以下は diaplays にかかっているように思えます (中断することなく表示する)。今の訳だと「中断することなく、注意を引く」とかかっているように読めます。読点を入れると多少よくなるかもしれません。

```suggestion
          <a class="role-reference" href="#alert">アラート</a>は、利用者のタスクを中断することなく、利用者の注意を引くように、簡潔で重要なメッセージを表示する要素です。動的にレンダリングされたアラートは、ほとんどのスクリーンリーダーで自動的にアナウンスされ、一部のオペレーティングシステムでは、アラート音が鳴る場合があります。現時点でスクリーンリーダーは、ページの読み込みが完了する前にページ上に存在するアラートを、利用者に通知しないことに注意することが重要です。
```